packages/grafana-ui/src/components/uPlot/utils.ts

```diff
@@ -62,18 +62,33 @@ export function preparePlotData(frame: DataFrame): AlignedData {
     seriesIndex++;
   }
 
+  const byPct = frame.fields[1].config.custom?.stacking?.mode === StackingMode.Percent;
```

Comment:
i guess we have the stacking config editor disabled, so this is just fudge to use the same mode for all?

Reply:
interesting -- then maybe we should move the bar-chart stacking option back to field config :thinking: and avoid the extra copy/shift thing